ietf/templates/iab_base.html

```diff
@@ -42,7 +42,11 @@
         {% block extra_css %}{% endblock %}
 
         {% analytical_head_bottom %}
-    <link href="{% static 'dist/main.css' %}" rel="stylesheet"></head>
+        <link href="{% static 'dist/main.css' %}" rel="stylesheet">
+        <link rel="icon" href="{% static '/img/cropped-iab-fav-32x32.png' %}" sizes="32x32">
+        <link rel="icon" href="{% static '/img/cropped-iab-fav-192x192.png' %}" sizes="192x192">
+        <link rel="apple-touch-icon" href="{% static '/wp-content/IAB-uploads/2022/07/cropped-iab-fav-180x180.png' %}">
```

Comment:
This location doesn't look right. I think this should be `/img/` similar to the above line.

Reply:
updated

Reply:
thanks.